### ISSUE TITLE
Is this grabber Legit?

### ISSUE THREAD

**Issue by mysteriouscitiesofgold** [NightfallGT/Mercurial-Grabber#28]
Is this real? Is there viruses or tojan? I downloaded it but I do not see any exe folder.

**Comment by NBDMatr1x** [comment]
yes, this is legit I have used it before and it works turn of your av as thats why the exe might not be there

**Comment by NadinesCrochet** [comment]
its legit
if you dont trust the builder its fukin open source take a look at the code
its just a false flag in the av
